nerdlandbot/commands/reminder.py

```diff
@@ -0,0 +1,97 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import *
+
+
+class Reminder(commands.Cog, name="Simple Reminder"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="reminder", aliases=["remind_me"], brief="reminder_brief", usage="reminder_usage", help="reminder_help")
+    async def reminder(self, ctx: commands.Context, time: typing.Optional[str] = None, *, message: typing.Optional[str] = None):
+        # If the user doesn't enter a time-format, let him know
+        msg = translate("reminder_err_no_int", await culture(ctx))
+        title = translate("reminder_err_title", await culture(ctx))
+        embed = discord.Embed(
+            description=msg,
+            color=NOTIFY_EMBED_COLOR
+        )
+        if time != None and time.isnumeric():
```

Comment:
When checking for None, try to use the `is` keyword, so `is not None` in this case